spec/middleware/parse_ebics_spec.rb

```diff
@@ -5,6 +5,7 @@
       builder.adapter :test, Faraday::Adapter::Test::Stubs.new do |stub|
         stub.post('/business_nok') {[ 200, {}, File.read( File.join( File.dirname(__FILE__), '..', 'fixtures', 'xml', 'ebics_business_nok.xml') )  ]}
         stub.post('/technical_nok') {[ 200, {}, File.read( File.join( File.dirname(__FILE__), '..', 'fixtures', 'xml', 'ebics_technical_nok.xml') )  ]}
+        stub.post('/technical_nok_unkown') {[ 200, {}, "The Server cannot process your request." ]}
```

Comment:
Curious why this works here, but a string is not a valid Rack response body in Ruby 1.9+, since there is no `#each` on string anymore.

I would wrap it in an array.